packages/shared/src/components/cards/ActionButtons.tsx

```diff
@@ -1,19 +1,26 @@
-import React, { ReactElement, ReactNode } from 'react';
+import React, { ReactElement, ReactNode, useRef } from 'react';
 import Link from 'next/link';
 import classNames from 'classnames';
 import dynamic from 'next/dynamic';
 import styles from './Card.module.css';
 import { Post } from '../../graphql/posts';
 import UpvoteIcon from '../../../icons/upvote.svg';
-import { getTooltipProps } from '../../lib/tooltip';
 import rem from '../../../macros/rem.macro';
 import InteractionCounter from '../InteractionCounter';
 import { QuaternaryButton } from '../buttons/QuaternaryButton';
 import CommentIcon from '../../../icons/comment.svg';
 import BookmarkIcon from '../../../icons/bookmark.svg';
 import { Button } from '../buttons/Button';
+import { getShouldLoadTooltip } from '../tooltips/LazyTooltip';
 
 const ShareIcon = dynamic(() => import('../../../icons/share.svg'));
+const Tooltip = dynamic(
+  () => import(/* webpackChunkName: "tooltip" */ '../tooltips/Tooltip'),
+);
```

Comment:
I actually think we don't have to lazy load it here, as it will always be visible from the start, what do you think?

Reply:
I think the part where it will help is it won't be included in the main build that needs to be loaded on the initial assets download. Though I'm not sure if this is overkill, I think you are right and it is probably better not to do it here anymore.